packages/ods/src/components/tooltip/src/components/ods-tooltip/ods-tooltip.tsx

```diff
@@ -28,6 +30,9 @@ export class OdsTooltip {
 
   @Method()
   public async hide(): Promise<void> {
+    if (this.isTooltipHover) {
```

Comment:
this prevents integrator to enforce closing (even if the user is on it), would be best to separate this behaviour from the exposed Method